app/channels/concerns/paper_trail_activity.rb

```diff
@@ -0,0 +1,17 @@
+# This module exists to assist with PaperTrail::Version properties in Channels.
+# Typically the controllers take care of this but this request process does not
+# utilize controllers. In controllers we set PaperTrail metadata in
+# ProjectScoped#info_for_paper_trail. For channels we will do it here as a
+# before_subscribe action.
+module PaperTrailActivity
```

Comment:
PaperTrail is an implementation detail, not part of the domain model. I'm sure you can find a better name for this Class and the job it does!

Reply:
I don't mind combining them. I'll make that change.

Reply:
Removed this module and combined it into `ProjectScopedChannels`.